src/app/resolvers/user/user.resolver.ts

```diff
@@ -1,13 +1,15 @@
 import {DataStore} from '../../services/data-store.service';
 import {EventEmitter} from '@angular/core';
 import {Injectable} from '@angular/core';
+import {JARoutes} from '../../routes/ja-routes';
+import {NavigationService} from '../../services/navigation.service';
 import {Resolve} from '@angular/router';
 import {User} from '../../models/user';
 import {UserProxy} from '../../services/proxy/user-proxy.service';
 
 @Injectable()
 export class UserResolver implements Resolve<User> {
-  private readonly storageAuthorizationData: string = 'authorizationData';
+  private readonly storageSessionKey: string = 'sessionData';
```

Comment:
Yes, might want to change the string back, but I thought session was a better name 👍 
@buren